internal/driver/fetch.go

```diff
@@ -110,7 +156,7 @@ func fetchProfiles(s *source, o *plugin.Options) (*profile.Profile, error) {
 // chunkedGrab fetches the profiles described in source and merges them into
 // a single profile. It fetches a chunk of profiles concurrently, with a maximum
 // chunk size to limit its memory usage.
-func chunkedGrab(sources []profileSource, fetch plugin.Fetcher, obj plugin.ObjTool, ui plugin.UI) (*profile.Profile, plugin.MappingSources, bool, int, error) {
+func chunkedGrab(sources []profileSource, fetch plugin.Fetcher, obj plugin.ObjTool, ui plugin.UI) (*profile.Profile, plugin.MappingSources, int, bool, error) {
```

Comment:
Oops -- I'd briefly gotten rid of the in return when I tried to move some of the error checking, realized that wouldn't work, and then added back the return incorrectly. 

I'll change it back.